### PR TITLE
Better phrasing and formatting for some docstrings

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -138,14 +141,18 @@ end
...
 
 ```math
-ℜ [ \\sum_{𝐤} û_{𝐤} L_x L_y ] \\,,
+ℜ [ \\sum_{𝐤} û_{𝐤} L_x L_y ] ,
 
 ```
 where ``û_{𝐤} =`` `uh` `` / (n_x e^{i 𝐤 ⋅ 𝐱₀})``. The elements of the vector ``𝐱₀`` are the
 left-most position in each direction, e.g., for a 2D grid `(grid.x[1], grid.y[1])`.
+
+When the input `uh` comes from a real-FFT transform, then `parsevalsum` takes care to
```
Comment (jbisits) [inline src/utils.jl:154]: When the input `uh` comes from a real-FFT transform, ~~then~~ `parsevalsum` takes care to..

I think the then is not needed here.

File: src/utils.jl [hunk 2]
```diff
@@ -138,14 +141,18 @@ end
...
+count twice the contribution from certain ``k``-modes.
```
Comment (jbisits) [inline src/utils.jl:155]: count the contribution from certain ``k``-modes twice.

Not necessary but maybe sounds better?

**Review by jbisits — COMMENTED** [review]
Small wording changes.

**Comment by navidcy** [comment]
Thanks! Done them.

**Review by jbisits — APPROVED** [review]

**Pull request merged**